src/scss/_objects.resources-tile.scss

```diff
@@ -0,0 +1,12 @@
+.resources-tile {
+  background-color: #fff;
```

Comment:
Try to always use Grommet variables for colors, such as `$background-color` here.